src/Phinx/Config/Config.php

```diff
@@ -49,6 +49,11 @@ class Config implements ConfigInterface
     protected $configFilePath;
 
     /**
+     * @var $phinx_version
+     */
+    public static $phinx_version = "0.4.1";
```

Comment:
I added it for the PhinxApplication to use, since the PHINX_VERSION wasn't visible. I'll revert these changes as per your other message though.